Autopilot/AttitudeManager/timer.c

```diff
@@ -36,10 +36,10 @@ void initTimer4(){
     T4CONbits.TGATE = 0; // Disable Gated Timer mode
     T4CONbits.TCKPS = 0b10; // Select 1:64 Prescaler
     TMR4 = 0x00; // Clear timer register
-    PR4 = 642; // Load the period value
-    IPC6bits.T4IP = 0x01; // Set Timer1 Interrupt Priority Level
-    IFS1bits.T4IF = 0; // Clear Timer1 Interrupt Flag
-    IEC1bits.T4IE = 1; // Enable Timer1 interrupt
+    PR4 = 625; // Load the period value
```

Comment:
I felt that T2_TICKS_TO_MSEC should be used for Timer2 only, in the event that the timers are set with different prescale values (could improve output response times). I will try to make a macro for this value, but that will still cause problems, as there's no simple way to convert from the prescaler value to a multiplier.